.vscode/settings.json

```diff
@@ -149,5 +151,11 @@
 
   // Whether the extension contributes a lint task to lint a whole workspace.
   "eslint.lintTask.enable": true,
+
+  // Avoids having to globally install ESLint.
+  "eslint.packageManager": "pnpm",
+
+  "eslint.enable": true,
+
   "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }]
```

Comment:
All these changes above were from a failed attempt to use pnpm as the main package manager.

Ultimately, i was unable to get the Prettier VSCode plugin to know where to find the Prettier module, so we are sticking w/ npm for now&hellip; I am going to leave all of that config there in case i try again later.